src/connect.rs

```diff
@@ -25,8 +25,8 @@ pub fn connect_to(peer_info: CrustInfo, send_after_connect: Option<WireMsg>) ->
         let transport_config = unwrap!(Arc::get_mut(&mut peer_cfg.transport));
         // TODO test that this is sent only over the uni-stream to the peer not on the uni
         // stream from the peer
-        transport_config.idle_timeout = ctx(|c| c.idle_timeout);
-        transport_config.keep_alive_interval = ctx(|c| c.keep_alive_interval);
+        transport_config.idle_timeout = ctx(|c| c.idle_timeout) * 1000;
```

Comment:
If they have changed to msec can we change the context struct and config struct to reflect this (so let that number denote msec instead of secs like now, and consequently not multiply by 1000 here)

0)
Chnage https://github.com/ustulation/using-quinn/blob/48c9799c69fc28f139ff0cb2e74bb17ae058011d/src/context.rs#L76 to
```
        idle_timeout_msec: u64,
```
1)
https://github.com/ustulation/using-quinn/blob/48c9799c69fc28f139ff0cb2e74bb17ae058011d/src/context.rs#L87 to
```
        keep_alive_interval_msec: u32,
```
2)
https://github.com/ustulation/using-quinn/blob/48c9799c69fc28f139ff0cb2e74bb17ae058011d/src/config.rs#L21 to
```
        pub idle_timeout_msec: Option<u64>,
```
and https://github.com/ustulation/using-quinn/blob/48c9799c69fc28f139ff0cb2e74bb17ae058011d/src/config.rs#L20 to
```
        /// The interval is in miliseconds. A value of 0 disables this feature.
```
3) same for `keep_alive_interval` and its doc-comments